analyze_results.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import json, os, re, sys, yaml
+from nis import match
```

Comment:
Hi @hvamstel , I think this inclusion isn't needed, right?

Reply:
No indeed, I didn't see this is my changes. I guess some of my tooling added it. I will remove it.